ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SchemaConverters.java

```diff
@@ -381,6 +383,7 @@ public SqlBaseType toSqlType(final Class<?> javaType) {
             .put(ParamTypes.LONG, SqlTypes.BIGINT)
             .put(ParamTypes.DOUBLE, SqlTypes.DOUBLE)
             .put(ParamTypes.TIMESTAMP, SqlTypes.TIMESTAMP)
+            .put(ParamTypes.LAMBDALITERAL, SqlTypes.LAMBDALITERAL)
```

Comment:
as mentioned above, I think what we should do here is not put it into the map and instead use:
```java
if (paramType instanceOf LambdaType) {
  return SqlTypes.lambda(toSqlType(paramType.input), toSqlType(paramType.output));
}
```